doc/source/drivers/vector/xodr.rst

```diff
@@ -0,0 +1,195 @@
+
+Convenient usage through docker image 
+-------------------------------------
+
+To use the XODR driver through a docker image, first build the image from the corresponding docker directory 
+    
+  ::
+
+    cd <gdal>/docker/ubuntu-full/
+    docker build -t gdal/xodr -f Dockerfile .
+
+For general usage information refer to `GDAL Docker images <https://github.com/OSGeo/gdal/tree/master/docker#usage>`__. Usage examples:
+
+- Use :program:`ogrinfo` to extract detailed information about a local `xodr` file by mounting your current working directory (`$PWD`) containing the file into the Docker container:
+  
+  ::
+
+    docker run --rm -v ${PWD}:/home -it gdal/xodr ogrinfo /home/<file>.xodr
+
+- Use :program:`ogr2ogr` to convert a local `xodr` file into any other supported OGR output format. The result will be automatically available in your host machine's working directory which is mounted into the container: 
+  
+  ::
+
+    docker run --rm -v ${PWD}:/home -it gdal/xodr ogr2ogr -f "GPKG" /home/<file>.gpkg /home/<file>.xodr
+
+
+Alternatively, you can run a docker container that enables using the XODR driver in an isolated workspace from within the container 
+    
+    ::
+
+      docker run --name <container_name> -it gdal/xodr /bin/bash
+
+
+General building notes
+----------------------
+
+Building of the driver as plugin is tested to work on
+
+* Ubuntu 24.04 using GCC
+* Windows 10 using GCC 13.1.0 (with MCF threads) + MinGW-w64 11.0.0 (MSVCRT runtime), which is obtainable from `WinLibs <https://winlibs.com/>`_.
+
+Ensure to meet the following driver dependencies:
+
+* PROJ
+* GEOS
+* libOpenDRIVE_ as shared library (built with CMake option ``-DBUILD_SHARED_LIBS=ON``)
+
+Then, after checking out GDAL sources with this driver extension, create the build directory:
+
+  ::
+
+    cd <gdal>
+    mkdir build
+    cd build
+
+From the build directory configure CMake to activate our XODR driver as plugin:
+
+  ::
+
+    cmake .. -DOGR_ENABLE_DRIVER_XODR_PLUGIN=TRUE -DOpenDrive_DIR=/path/to/libOpenDRIVE/installdir/cmake/
+
+.. note:: The :file:`cmake/` path is usually automatically created when installing libOpenDRIVE and contains the necessary configuration files for inclusion into other project builds.
+
+Now, build GDAL and install it:
+
+  ::
+
+    cmake --build .
+    cmake --build . --target install
+
+Afterwards you will find a new shared library file :file:`{path/to/GDAL/installdir}/lib/gdalplugins/ogr_XODR`.
+
+Verifying a successful build
+++++++++++++++++++++++++++++
+
+Check if XODR driver is found:
+
+  ::
+    
+    cd <gdal>/build/
+    ./apps/ogrinfo --formats
+
+This should print a list of supported OGR formats, including ``XODR`` in the first row:
+
+  ::
+
+    Supported Formats:
+      XODR -vector- (rov): OpenDRIVE - Open Dynamic Road Information for Vehicle Environment
+      PCIDSK -raster,vector- (rw+v): PCIDSK Database File       
+      PDS4 -raster,vector- (rw+vs): NASA Planetary Data System 4
+      ...
```

Comment:
More simple to just run "ogrinfo --format XODR". Furthermore XODR might not always be in the first row